README.md

```diff
@@ -40,3 +38,18 @@ question.
 It is licensed under the [MPL 2.0](https://www.mozilla.org/en-US/MPL/2.0/). All
 contributions retain ownership by their original author, but must also be
 licensed under the MPL 2.0 to be merged by us.
+
+### Setting up for development
+
+Stratisd runs as root, and requires access to the D-Bus system bus. Thus in
+order to work properly, a D-Bus conf file must exist to grant access, either
+installed by distribution packaging; or manually, by copying `stratisd.conf`
+to `/etc/dbus-1/system.d/`.
+
+Stratisd requires Rust 1.15.1+ and Cargo to build. These may be available via
+your distribution's package manager. If not, [Rustup](https://www.rustup.rs/)
+is available to install and update the Rust toolchain.
+
+Once toolchain is in place, run `cargo build` to build, and then run the
```

Comment:
stratisd need to be run as root. Running through Cargo as root (e.g. `sudo cargo run -- --debug`) recompiles everything as root: uses /root/.cargo/config, generated files are owned by root etc., which is not desired. I think it's better to compile as a user, then only run as root.